client/header/activity-panel/index.js

```diff
@@ -93,10 +93,14 @@ class ActivityPanel extends Component {
 		} ) );
 	}
 
-	handleClickOutside() {
+	handleClickOutside( event ) {
 		const { isPanelOpen, currentTab } = this.state;
+		const isClickOnModalOrSnackbar =
```

Comment:
Is there any chance `event` or `event.target` could be undefined here? I suppose we could add check and clean up the code a bit since event.target.closest is used 2x.